ThunderCloud/OrderedListItem.swift

```diff
@@ -29,8 +29,8 @@ open class OrderedListItem: ListItem {
 		super.configure(cell: cell, at: indexPath, in: tableViewController)
 		
 		guard let numberCell = cell as? NumberedViewCell else { return }
-		
 		numberCell.numberLabel.text = number
+        numberCell.numberLabel.font = ThemeManager.shared.theme.dynamicFont(ofSize: 28, textStyle: .title2, weight: .medium)
```

Comment:
Will re-indent all files once approved so that it's easier to see the changes I've made 😊